content/lessons/chapter-4/public-key-3.tsx

```diff
@@ -98,15 +90,26 @@ export default function PublicKey3({ lang }) {
   const t = useTranslations(lang)
 
   const [language, setLanguage] = useState(config.defaultLanguage)
-
+  const [hiddenRange, setHiddenRange] = useState([1, 0, 1, 0])
   const handleSelectLanguage = (language: string) => {
     setLanguage(language)
   }
 
+  useEffect(() => {
+    if (language === 'python') {
+      setHiddenRange([1, 0, 123, 0])
+    }
+    if (language === 'javascript') {
+      setHiddenRange([1, 1, 1, 1])
+    } else {
+      setHiddenRange([])
```

Comment:
Is this what fixes the hiddenRange switch? does it need to be on column 1 instead of 0 for any reason?

Reply:
nope removed that.